### PR TITLE
update owners to include kn plugin approvers

### PR DESCRIPTION
**Pull request by csantanapr**
update owners to include kn plugin approvers, this will allow better management of users to handle the repo

### CODE REVIEW

**Comment by csantanapr** [comment]
/hold
waiting for github team to be created first https://github.com/knative/community/pull/695

**Comment by csantanapr** [comment]
this one is merge now https://github.com/knative/community/pull/695
/unhold

**Comment by csantanapr** [comment]
It looks like OWNERS_ALIASES file is missing `kn-plugin-quickstart-approvers`

The update community github action runs twice a day but didn't do a PR to update OWNERS_ALIASES

**Review by psschwei — APPROVED** [review]
/lgtm

**Pull request merged**
